public/app/views/partials/users/welcome.jade

```diff
@@ -0,0 +1,5 @@
+div.page-header.text-center
+  header.business-header
+    .container
```

Comment:
Oh, I didn't notice that in home.jade. This creates an empty div with the .contianer class (jade defaults to creating a div when no other thing is specified before  a class or id). Would you mind removing it from home.jade as well?